src/packages/@ncigdc/modern_components/ProjectPrimarySitesTable/ProjectPrimarySitesTable.js

```diff
@@ -0,0 +1,193 @@
+/* @flow */
+
+import React from 'react';
+import { compose, branch, mapProps, withState, renderNothing } from 'recompose';
+import { connect } from 'react-redux';
+import { Row } from '@ncigdc/uikit/Flex';
+import LocalPaginationTable from '@ncigdc/components/LocalPaginationTable';
+import tableModels from '@ncigdc/tableModels';
+import Table, { Tr, Td } from '@ncigdc/uikit/Table';
+import styled from '@ncigdc/theme/styled';
+import SearchIcon from 'react-icons/lib/fa/search';
+import Input from '@ncigdc/uikit/Form/Input';
+import CloseIcon from '@ncigdc/theme/icons/CloseIcon';
+import { trim } from 'lodash';
+import { withTheme } from '@ncigdc/theme';
+
+const Header = styled(Row, {
+  padding: '1rem',
+  color: ({ theme }) => theme.greyScale7 || 'silver',
+  fontSize: '1.7rem',
+});
+
+const PrimarySitesTable = ({ data, tableInfo, projectId }) => (
+  <Table
+    id="project-primary-site-table"
+    headings={tableInfo
+      .filter(x => !x.subHeading)
+      .map(x => <x.th key={x.id} />)}
+    subheadings={tableInfo
+      .filter(x => x.subHeading)
+      .map(x => <x.th key={x.id} />)}
+    body={
+      <tbody>
+        {data.length >= 1 ? (
+          data.map((primarySite, i) => {
+            return (
+              <Tr key={i} index={i}>
+                {tableInfo
+                  .filter(x => x.td)
+                  .map(x => (
+                    <x.td
+                      key={x.id}
+                      primarySite={primarySite}
+                      projectId={projectId}
+                    />
+                  ))}
+              </Tr>
+            );
+          })
+        ) : (
+          <Tr>
+            <Td style={{ padding: '10px' }}>
+              There are no results for that query
+            </Td>
+          </Tr>
+        )}
+      </tbody>
+    }
+  />
+);
+
+export default compose(
+  // setDisplayName('ProjectPrimarySitesTablePresentation'),
+  branch(
+    ({ viewer }) =>
+      !viewer.projects.hits.edges ||
+      viewer.projects.hits.edges[0].node.primary_site.length <= 1,
```

Comment:
it should render nothing if only 1?

Reply:
Yeah. The table is only supposed to show if there are multiple primary sites.

Reply:
can that be changed to `< 2` then